lib/riot_js/rails/processors/sprockets_processor_v3.rb

```diff
@@ -18,7 +18,8 @@ def call(input)
 
       def self.register_self(config)
         config.assets.configure do |env|
-          env.register_engine '.tag', self, mime_type: 'application/javascript'
```

Comment:
The same issue exists here:

    gems/riot_js-rails-0.4.0 ➲ ag register_engine
    lib/riot_js/rails/processors/sprockets_processor_v2.rb
    8:        app.assets.register_engine '.tag', self

    lib/riot_js/rails/processors/sprockets_processor_v3.rb
    21:          env.register_engine '.tag', self, mime_type: 'application/javascript'

    lib/riot_js/rails/railtie.rb
    16:          app.assets.register_engine '.haml', ::Tilt::HamlTemplate